app/assets/javascripts/admin/api_flow_types.js

```diff
@@ -97,25 +97,26 @@ export type APIAnnotationType = {
   +tracingTime: null,
   +tags: Array<string>,
 };
+export type APITaskTypeType = {
+  +id: string,
+  +summary: string,
+  +description: string,
+  +team: string,
+  +settings: {
+    +allowedModes: ["orthogonal", "oblique", "flight"],
```

Comment:
This means that allowedModes always is this exact array? Shouldn't it be an array of the union? Also can we reuse types from oxalis? It reminds of a restrictions type or something similar.